src/utils/error.ts

```diff
@@ -1,5 +1,6 @@
 import { PlainObject } from "is_what"
 import { undent } from "utils"
+import { useFlags } from "hooks";
```

Comment:
this causes a cyclic dependency (which is why those tests are failing).

Reply:
Oh thank you so much. I've been sitting here scratching my head trying to figure it out since everything is running fine for me.